src/module.c

```diff
@@ -258,14 +258,24 @@ typedef struct _modstack_t {
 
 static jl_binding_t *jl_get_binding_(jl_module_t *m JL_PROPAGATES_ROOT, jl_sym_t *var, modstack_t *st);
 
+static inline jl_module_t *module_usings_getidx(jl_module_t *m JL_PROPAGATES_ROOT, size_t i) JL_NOTSAFEPOINT;
```

Comment:
I'm surprised it doesn't warn the static inline method isn't defined by the end of the scope, but doesn't seem to?

Reply:
No, this is explicitly fine. You can define a static inline in a header, it's just an error if you use it without providing a definition.